.changeset/chilled-jeans-march.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/task': major
+---
+
+Adds the `'afterUpdate'` option for `autoRun` to Task, and runs tasks by default in `hostUpdate()` instead of `hostUpdated()`.
```

Comment:
Is it the case that users that are broken can run with `afterUpdate` timing to get the old behavior, but that they'll get better performance with the new timing? If so, I think this is a good thing to call out in the changelog